jsonrpc/src/methods/history.json

```diff
@@ -287,8 +287,11 @@
   },
   {
     "name": "portal_historyOffer",
-    "summary": "Send the provided content item to interested peers. Clients may choose to send to some or all peers.",
+    "summary": "Send an OFFER request with given ContentKey, to the designated peer and wait for a response.",
```

Comment:
If we return a boolean, we should edit this description because we are not waiting for the response anymore.

Reply:
As mentioned, in other comment, this implementation is with the idea of waiting for response + potential uTP transfer.